tests/get_or_create/tests.py

```diff
@@ -441,24 +441,38 @@ def test_updates_in_transaction(self):
         while it holds the lock. The updated field isn't a field in 'defaults',
         so update_or_create() shouldn't have an effect on it.
         """
+
+        lock_status = {'has_grabbed_lock': False}
```

Comment:
I had originally used a `global`, but usually try to avoid it if possible.  I have pushed the change to use `global` instead.